<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>393</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #FFF8DC;  /* Çok açık sarımsı krem */
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="4">
    <widget class="QWidget" name="frameReminderOptions" native="true">
     <property name="styleSheet">
      <string notr="true">QRadioButton {
    background-color: #ffffff;
    border: 2px solid #bbb;
    border-radius: 12px;
    padding: 12px;
    font-size: 15px;
    font-weight: bold;
    color: #444;
    min-width: 150px;
    text-align: center;
    qproperty-iconSize: 20px 20px;
}

QRadioButton::indicator {
    width: 0px;  /* klasik yuvarlağı gizle */
}

QRadioButton:hover {
    background-color: #f0f8ff;
    border: 2px solid #88c;
}

QRadioButton:checked {
    background-color: #d0f0c0;
    border: 2px solid #4CAF50;
    color: #2b4;
}
</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QRadioButton" name="radioButton3">
        <property name="styleSheet">
         <string notr="true">QRadioButton {
    background-color: #ffffff;
    border: 2px solid #bbb;
    border-radius: 12px;
    padding: 12px;
    font-size: 15px;
    font-weight: bold;
    color: #444;
    min-width: 150px;
    text-align: center;
    qproperty-iconSize: 20px 20px;
}

QRadioButton::indicator {
    width: 0px;  /* klasik yuvarlağı gizle */
}

QRadioButton:hover {
    background-color: #f0f8ff;
    border: 2px solid #88c;
}

QRadioButton:checked {
    background-color: #d0f0c0;
    border: 2px solid #4CAF50;
    color: #2b4;
}

</string>
        </property>
        <property name="text">
         <string>         45 dakika</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioButton1">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
    background-color: #ffffff;
    border: 2px solid #bbb;
    border-radius: 12px;
    padding: 12px;
    font-size: 15px;
    font-weight: bold;
    color: #444;
    min-width: 150px;
    text-align: center;
    qproperty-iconSize: 20px 20px;
}

QRadioButton::indicator {
    width: 0px;  /* klasik yuvarlağı gizle */
}

QRadioButton:hover {
    background-color: #f0f8ff;
    border: 2px solid #88c;
}

QRadioButton:checked {
    background-color: #d0f0c0;
    border: 2px solid #4CAF50;
    color: #2b4;
}

</string>
        </property>
        <property name="text">
         <string>         15 Dakika</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="radioButton2">
        <property name="styleSheet">
         <string notr="true">QRadioButton {
    background-color: #ffffff;
    border: 2px solid #bbb;
    border-radius: 12px;
    padding: 12px;
    font-size: 15px;
    font-weight: bold;
    color: #444;
    min-width: 150px;
    text-align: center;
    qproperty-iconSize: 20px 20px;
}

QRadioButton::indicator {
    width: 0px;  /* klasik yuvarlağı gizle */
}

QRadioButton:hover {
    background-color: #f0f8ff;
    border: 2px solid #88c;
}

QRadioButton:checked {
    background-color: #d0f0c0;
    border: 2px solid #4CAF50;
    color: #2b4;
}
</string>
        </property>
        <property name="text">
         <string>         30 dakika</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="radioButton4">
        <property name="styleSheet">
         <string notr="true">QRadioButton {
    background-color: #ffffff;
    border: 2px solid #aaa;
    border-radius: 10px;
    padding: 10px;
    font-size: 14px;
    font-weight: bold;
    color: #333;
}

QRadioButton::indicator {
    width: 0px;  /* yuvarlak noktasını gizle */
}

QRadioButton:checked {
    background-color: #d0eaff;
    border: 2px solid #3399ff;
    color: #000;
}
</string>
        </property>
        <property name="text">
         <string>        60 dakika</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="buttonBack">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 2px solid #ccc;
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
    font-size: 14px;
    color: #333;
}
QPushButton:hover {
    background-color: #f0f0f0;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_hatirlatmaOnayi">
        <property name="styleSheet">
         <string notr="true"> background-color: #e6ffe6;
 color: #2d862d;
 border: 2px solid #5cd65c;
 border-radius: 10px;
 padding: 8px;
 font-weight: bold;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_gorev">
        <property name="styleSheet">
         <string notr="true">QLabel {
    background-color: #f5f5f5;
    border: 2px solid #aaa;
    border-radius: 10px;
    padding: 10px;
    color: #333;
    font-size: 16px;
    font-weight: bold;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="startButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
          <italic>false</italic>
          <bold>true</bold>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: white;
    border: 1px solid #ccc;
    border-radius: 10px;
    font-weight: bold;
    color: black;
    padding: 8px;
}

QPushButton:hover {
    background-color: #f0f0f0;
    border: 1px solid #999;
}
</string>
        </property>
        <property name="text">
         <string>BAŞLA</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="button3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: white;
    border: 1px solid #ccc;
    border-radius: 10px;
    font-weight: bold;
    color: black;
    padding: 8px;
}

QPushButton:hover {
    background-color: #f0f0f0;
    border: 1px solid #999;
}


</string>
        </property>
        <property name="text">
         <string>NEFES EGZERSİZLERİ</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../Downloads/breathing.png</normaloff>../../../Downloads/breathing.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="button4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: white;
    border: 1px solid #ccc;
    border-radius: 10px;
    font-weight: bold;
    color: black;
    padding: 8px;
}

QPushButton:hover {
    background-color: #f0f0f0;
    border: 1px solid #999;
}
</string>
        </property>
        <property name="text">
         <string>GÖZ DİNLENDİRME</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../Downloads/eyes.png</normaloff>../../../Downloads/eyes.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="button2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: white;
    border: 1px solid #ccc;
    border-radius: 10px;
    font-weight: bold;
    color: black;
    padding: 8px;
}

QPushButton:hover {
    background-color: #f0f0f0;
    border: 1px solid #999;
}
</string>
        </property>
        <property name="text">
         <string>SU İÇME</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../Downloads/mineral-water.png</normaloff>../../../Downloads/mineral-water.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="button1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>299</width>
          <height>29</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: white;
    border: 1px solid #ccc;
    border-radius: 10px;
    font-weight: bold;
    color: black;
    padding: 8px;
}

QPushButton:hover {
    background-color: #f0f0f0;
    border: 1px solid #999;
}

</string>
        </property>
        <property name="text">
         <string>DİK DURMA</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../Downloads/orthopedics.png</normaloff>../../../Downloads/orthopedics.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLabel" name="labelWelcome">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>26</pointsize>
          <italic>true</italic>
          <hintingpreference>PreferDefaultHinting</hintingpreference>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 600 italic 26pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>HOŞGELDİNİZ</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
